doc/specs/staffPage.md

```diff
@@ -0,0 +1,161 @@
+# Staff Page
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+4. Data model
+
+## Motivation
+
+>The staff team is the core of the smooth operation of the site. The staff page is a list of the site's staff members for the user's convenience.
+
+## Business Rules
+
+> * User classes must have an option to be presented on the staff page.
+> * Users with the class management permission must be able to toggle if a user class is displayed on the staff page.
+> * User classes with dedicated staff page descriptions must get their own sections on the staff page (i.e. forum moderators).
```

Comment:
Only if they have a description? or should their description be shown alongside the class?

(Also I'm not really sure this is a business rule as much as it is a functional requirement)